app/jobs/add_live_result_job.rb

```diff
@@ -4,8 +4,22 @@ class AddLiveResultJob < ApplicationJob
   self.queue_adapter = :shoryuken if WcaLive.sqs_queued?
   queue_as EnvConfig.LIVE_QUEUE if WcaLive.sqs_queued?
 
+  # params: { results, round_id, user_id, entered_by }
   def perform(params)
-    ActionCable.server.broadcast(WcaLive.broadcast_key(params[:round_id]),
-                                 { attempts: params[:results], user_id: params[:user_id] })
+    results = params[:results]
+    attempts = results.map.with_index(1) { |r, i| LiveAttempt.build(result: r, attempt_number: i) }
+    round = Round.find(params[:round_id])
+    event = round.event
+    format = round.format
+
+    r = Result.build({ value1: results[0], value2: results[1] || 0, value3: results[2] || 0, value4: results[3] || 0, value5: results[4] || 0, event_id: event.id, round_type_id: round.round_type_id, format_id: format.id })
```

Comment:
Why are you using `build` instead of `new` here?

Reply:
when using new it gives me an error saying `Found extra argument. Required 0 ` I think because there is no initializer?